app/models/storage_system.rb

```diff
@@ -0,0 +1,141 @@
+class StorageSystem < ApplicationRecord
+  # include_concern 'Operations'
+
+  include NewWithTypeStiMixin
+  include ProviderObjectMixin
+  include AsyncDeleteMixin
+  include AvailabilityMixin
+  include SupportsFeatureMixin
+  include CustomActionsMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id,
+             :class_name => "ExtManagementSystem", :inverse_of => :storage_systems
```

Comment:
After we added the :inverse_of, we started getting an error because ext_management_system doesn't have the storage_systems field. Instead we had put it in BlockMixin that our manager utilizes. 
Do we need to this relationship polymorphic? It would be wasteful because it would always point to ExtManagementSystem.